HomeWork_One/Quiz_Template/src/components/inputs/SelectTime.tsx

```diff
@@ -0,0 +1,28 @@
+import './Select.css'
+import {SettingsContext} from "../Context.tsx";
+import {useContext} from "react";
+import { v4 as uuidv4 } from 'uuid'
+
+function SelectTime() {
+    const {timeOptions} = useContext(SettingsContext)
+    console.log(timeOptions)
+    return(
+        <>
+            <div className='input-container'>
+                <label htmlFor="select" className="select-label">Time</label>
+                <select name="select-setting" className="select">
+                    {
+                        timeOptions ? timeOptions.map((el)=>{
+                            return (
+                                <option key={el.id}>{el.value}</option>
+                            )
+                        }) : <option value="">template</option>
+                    }
+                </select>
+            </div>
+
+        </>
+    )
```

Comment:
this UI can be extracted into a separate common component
because it repeats here and in Select